www/index.php

```diff
@@ -1,12 +1,16 @@
 <?php
 use UNL\Templates\Templates;
 use Endroid\QrCode\QrCode;
+use Ramsey\Uuid\Uuid;
 
 require_once __DIR__ . '/../config.inc.php';
 
 $lilurl = new lilURL(MYSQL_HOST, MYSQL_USER, MYSQL_PASS, MYSQL_DB);
 $lilurl->setAllowedProtocols($allowed_protocols);
 $lilurl->setAllowedDomains($allowed_domains);
+if (defined('GA_ACCOUNT')) {
+    $lilurl->setGaAccount(GA_ACCOUNT);
```

Comment:
In our case, I'm guessing that `GA_ACCOUNT` will be set server side and point to our main property. How do site owners with custom properties consume these analytics? For example if music.unl.edu has their own GA account with tracking, how do they get access to this data? Or are they even supposed to?